src/app/(with-nav-footer)/(mypage)/my-activities/add-activities/_components/AddActivities.tsx

```diff
@@ -0,0 +1,176 @@
+    { label: '투어', onClick: () => {} },
+    { label: '관광', onClick: () => {} },
+    { label: '웰빙', onClick: () => {} },
+  ];
+
+  const methods = useForm<CreateActivityParams>({
+    mode: 'onChange',
+    resolver: zodResolver(createActivitySchema),
+    defaultValues: {
+      title: '',
+      category: '',
+      description: '',
+      address: '',
+      schedules: [],
+      subImageUrls: [],
+      bannerImageUrl: '',
+    },
+  });
+
+  const {
+    register,
+    control,
+    handleSubmit,
+    setValue,
+    reset,
+    formState: { errors, isValid },
+  } = methods;
+
+  const [bannerImageUrl, setBannerImageUrl] = useState<string>('');
+  const [subImageUrls, setSubImageUrls] = useState<string[]>([]);
+
+  const onSubmit = (data: CreateActivityParams) => {
+    const finalData = {
+      ...data,
+      bannerImageUrl,
+      subImageUrls,
+    };
+
+    createActivity(finalData);
+    toast.success('내 체험 등록 성공');
+    reset();
+  };
+
+  return (
+    <FormProvider {...methods}>
+      <form
+        className='space-y-6'
+        onSubmit={handleSubmit(onSubmit, (invalidData) => {
+          console.log(invalidData);
+        })}
+      >
+        <div className='flex items-center justify-between'>
+          <h2 className='text-2xl font-bold'>내 체험 등록</h2>
+          <Button type='submit' disabled={isPending || !isValid} className='px-[20px] py-[11px]'>
+            {isPending ? '등록 중...' : '등록하기'}
+          </Button>
+        </div>
+
+        <Input
+          label='제목'
+          placeholder='제목을 입력하세요'
+          {...register('title')}
+          error={errors.title?.message as string}
+        />
+
+        <label className='text-black-100 inline-flex items-center gap-1 text-lg font-medium'>카테고리</label>
+        <Controller
+          control={control}
+          name='category'
+          render={({ field }) => (
+            <FilterDropdown
+              label='카테고리'
+              options={categoryOptions}
+              onSelect={(option) => field.onChange(option?.label || '')}
+              icon={arrowFilterDropdown2}
+              buttonClassName='border lg:w-[792px] w-[343px] md:w-[429px] text-black-100 border-gray-800 rounded-lg md:justify-between px-[15px] py-[15px]'
+              dropdownClassName='rounded-xl lg:w-[792px] w-[343px] md:w-[429px] border border-gray-300 bg-white drop-shadow-sm'
+              optionClassName='text-md md:text-lg h-[41px] lg:w-[792px] w-[343px] md:w-[429px] leading-[41px] md:h-[58px] md:leading-[58px]'
+              includeAllOption={false}
+              iconVisibleOnMobile={false}
+              value={field.value}
+            />
+          )}
+        />
+        {errors.category && <p className='text-sm text-red-500'>{errors.category.message}</p>}
+
+        <Input
+          label='설명'
+          placeholder='설명을 입력해주세요'
+          className='h-[260px]'
+          {...register('description')}
+          error={errors.description?.message as string}
+        />
```

Comment:
```suggestion
      <label className='text-black-100 inline-flex items-center gap-1 text-lg font-medium'>설명</label>
       <textarea className='h-[346px] w-full rounded-[4px] border border-gray-800 px-4 py-2' placeholder='설명' />
```
textarea를 사용해보는건 어떨까용? 인풋으로 등록하니 !/~/. 같은 특수문자기호를 썼을 경우 / 자음만 써져있을 경우에 문자열로 지원되지 않아서 에러로 취급이 되네요! 그리고 줄바꿈이 안되는 문제가 있습니다 ㅠㅠ